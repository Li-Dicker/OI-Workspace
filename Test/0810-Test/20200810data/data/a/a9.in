10
197
87084 21376 79911 78574 29757 95297 48211 35765 35845 98613 81223 35628 57550 78699 92434 68024 58724 57758 72826 18256 70340 45539 34619 91253 10004 22926 52868 50405 53077 93953 51862 69277 77274 58264 82631 8017 47026 57880 86438 95172 20000 26740 16638 92074 77490 14555 41448 84509 34076 77363 73990 29297 29222 83518 52171 52457 95470 26654 45830 5659 49805 94575 255 34761 59220 54279 28845 16736 59529 10459 35431 26284 32490 48368 1644 25534 67296 36685 25850 6074 14785 39015 54874 42267 15917 16524 22678 20498 33117 33437 34853 56131 31515 54344 2005 75517 77238 59604 44263 75586 16768 59243 65596 92128 3673 66950 49654 27355 44906 78542 38214 22979 18902 30497 49383 41274 24270 70778 24742 69959 61503 20460 8763 37491 50354 13210 62941 19383 93720 76957 7115 39027 32584 99135 96493 44397 88707 7374 95535 11995 32836 50429 21951 5708 20390 66099 11160 25122 7387 44304 53334 64108 43960 53577 45087 1800 9322 93055 93466 12278 90809 53170 75877 84671 23697 46461 65757 13351 86844 4167 16555 188 75035 90859 66259 73924 37298 71431 72428 5629 28028 59943 19836 6935 53494 78884 60360 51962 62247 4422 92652 49273 96448 94326 27098 34212 71178 
31569 9645 14824 7330 14662 19872 17702 4004 8033 39005 31362 21378 16608 12978 9504 16958 48969 44659 40175 14260 10472 3091 14776 41208 8515 7793 861 15960 43505 2821 39797 31799 42815 49694 33329 32386 8701 19379 7376 38315 25224 17271 45998 1299 20747 8694 9558 43567 722 13773 19344 21274 45304 17506 49662 12906 38391 37699 47891 47176 36235 6088 19636 20727 48608 19742 19542 9497 24241 49687 14052 2938 35481 36547 33875 1954 38221 22243 10903 16237 19949 35233 48663 14933 5287 29184 49155 49836 36358 17497 30935 40606 29401 28493 41122 30291 47579 37559 2124 4392 39967 18219 10837 3258 32285 8028 17084 30801 33543 29317 47031 25852 40492 2664 26419 13360 38735 49364 21525 45658 21759 17014 24477 35751 31977 24706 29142 22420 5618 20937 47294 45189 16214 24496 17418 7050 40491 24008 33226 31436 16384 16679 9339 22616 24402 28326 19526 32952 31844 20658 29555 26777 39078 46473 5817 3489 4660 43188 11953 7647 12056 20411 25779 14844 8530 33872 47603 16580 8395 16034 19926 14432 9188 16254 4736 7043 27241 7264 24576 44947 6393 38771 12582 27506 20072 8464 16673 36441 12660 41722 33047 44784 42085 21006 680 36993 41089 
197
63546 91878 83294 30602 97101 29089 47049 8909 22283 30367 88792 23098 95948 48937 97414 29823 11288 74849 72876 96559 2442 91576 13637 45554 96854 44221 38250 95888 8808 4300 3614 60209 63828 52894 74313 3388 88610 74613 83493 1203 27689 65758 48239 10552 67596 90248 16721 36340 97884 63209 82590 54459 66100 87355 7549 2754 93276 8648 56490 84516 66516 36524 12342 46654 95403 59790 81560 77889 48958 42292 79347 62355 8443 63853 58513 94424 67177 48468 15427 99284 7446 3331 35490 95469 11906 1481 22464 54734 27700 96250 28536 3847 31368 70884 1935 97728 16329 99040 878 55441 39950 896 35867 15521 95634 15224 27974 78468 21067 30914 67797 6386 40696 91570 44040 96356 75398 50428 71211 67295 37713 36865 65889 38124 72406 57341 56554 12734 99377 39032 85695 21025 70854 70024 105 64108 33343 90679 74640 91270 11563 59601 64488 34062 14744 38972 30352 89802 72356 93075 45167 17572 42825 78409 68716 48459 43955 24424 38888 77967 71532 66244 67997 15502 34559 56272 47289 64039 83743 46324 41337 36585 2870 46904 45318 9568 26931 4064 70740 14748 38316 76981 93309 36921 80621 17970 46851 86327 90711 59921 98651 25499 34955 80163 30615 22156 5884 
41357 42125 6366 21103 13711 7828 17057 7754 9912 28317 39213 9543 5218 22136 25400 14719 572 31192 18226 40142 44146 222 8894 7414 42617 5354 1195 43355 31 7740 19621 44072 39685 39604 20985 32745 1612 15897 9139 2590 3845 32044 2713 39161 12970 414 1616 6741 9475 32905 43061 11717 39755 2895 27297 21012 49710 43068 44077 21997 6428 26665 11779 40968 8969 10051 419 41945 24904 28881 27045 25589 19082 34468 20745 45231 33165 33985 43915 46701 4137 41082 46299 28305 41757 21226 42707 37498 34961 44002 12664 14924 45927 5069 8419 5671 43988 9096 41833 7413 11113 48972 41061 47561 34504 8089 25997 4755 47455 24676 42569 16093 19311 24465 47847 43130 42492 12917 21619 26962 18647 4819 15507 18767 33971 36180 41211 7959 35007 7877 1714 7911 23629 44201 15261 25505 29017 9571 27714 49250 15802 879 39577 46395 11496 22452 28360 1461 49861 9114 18088 42285 14837 4927 12549 20553 28103 25946 48033 12770 14384 26140 17929 49034 16511 10525 27866 27114 37738 47298 11701 37066 24673 19240 28328 31520 38914 3786 6049 48594 37553 48870 40680 2186 38994 36176 12338 32447 3069 26895 9136 8810 22388 41255 46966 31586 48664 
197
85223 51725 54790 26435 80756 99329 39598 12898 53683 41532 26890 53185 71936 15339 60761 75514 75421 7043 7657 31018 91176 77409 34290 77809 78145 54372 54714 84879 36289 71498 40761 12608 28502 49531 99255 19758 10416 22850 78805 10266 75705 1509 93046 93721 2507 53810 92775 66377 55606 61365 40235 74039 49483 44317 28948 20558 27587 64116 75404 61545 30230 82967 39574 8516 14798 53819 19735 79523 66307 46799 31571 57701 67018 7892 33059 50122 36157 3676 96638 3205 43837 53761 43959 26141 194 83654 41393 78370 5220 57026 91701 79240 13304 14107 42764 25630 27425 44868 92516 45948 32336 63379 797 7361 34807 23608 43685 51998 6689 70895 57898 58635 33322 71010 34376 6335 95908 56314 86085 42669 93640 43750 41322 39635 34268 51347 14688 15375 72849 86582 74106 79009 36547 79506 38548 26090 99270 36411 70838 48740 65983 84535 83556 90607 37024 12917 43442 10516 10671 44626 23280 95251 22919 11801 80076 64158 80889 28589 32700 7727 1616 54181 8019 51707 57788 13412 11576 67729 72632 38633 78505 69956 13391 86681 62538 70057 18746 5787 58981 93839 31547 43930 96846 29223 10120 6274 9519 92309 36386 28882 67893 83516 18512 78784 86915 60162 61929 
37568 3975 21214 6355 12395 34845 35340 27313 35352 31753 46242 8140 49210 19812 32291 31030 408 24197 47819 34448 47343 41588 992 19066 5942 47114 22914 47836 5546 13709 5870 48079 11431 21118 40315 4983 24058 29346 30065 12645 46119 35341 18222 5564 46311 48465 13739 29645 13524 31299 13216 28869 21037 25476 17831 26625 23794 2817 43033 3781 3278 45031 26865 47946 46510 21972 36204 18061 46473 27290 9585 21329 22744 4213 22397 33225 34807 37799 24501 21712 11584 28682 22750 24847 35750 19009 32325 16016 17773 13825 44419 7402 32679 31170 20392 14542 22693 31761 32193 21165 41534 29696 25819 35807 2663 29073 5158 34964 41793 33684 9979 3451 34807 14315 17662 34819 49587 9787 22257 44126 41877 2884 46211 42516 26354 17324 10781 44787 13943 35311 4687 38428 7045 18678 2761 11960 25036 14430 4992 34576 42967 24080 6199 39711 40615 23722 11218 21197 15863 32320 14014 299 6260 19020 12755 3966 15514 14198 40989 16970 1847 21963 45929 39193 45603 26324 9565 46347 24181 988 38642 10370 13724 12787 19559 26711 30226 1755 33661 44720 20639 47673 47799 27307 16340 30164 38027 34874 34185 16659 40160 43546 5551 38963 18207 33476 14309 
200
66109 52730 42008 71047 80533 99292 34161 44804 63971 138 47370 9864 91507 39740 10384 39805 71369 25125 11759 19537 74092 70135 80228 98862 1922 56080 89373 84574 60612 60371 34381 74479 43725 46286 3720 92158 9283 54517 44790 93027 98312 18730 99205 35016 28067 72022 12823 35705 86017 99 18705 20112 91766 21441 19185 69153 76552 96648 4768 26945 36497 93195 85575 54105 69954 21478 13370 67562 71721 2731 57352 12164 28909 78821 65463 96897 41945 58936 18950 27700 84032 92996 38057 7268 42144 5584 65692 62585 615 13685 57005 43761 39924 53635 71254 53015 23444 18844 35079 36527 66599 60495 18264 7386 7422 93960 17725 71655 34310 17762 20485 92291 26213 87010 74899 94020 94350 11172 41592 92522 4564 99533 81291 50433 13028 68845 46750 49161 2596 2053 18271 16899 40151 21249 28160 1951 96829 94087 38922 51814 27785 49403 84317 22366 9041 54009 17939 42520 38940 74293 55702 54271 94019 94008 41029 77088 48784 79011 28354 48674 48218 47790 22011 46247 25595 32623 674 35492 14321 94432 62809 6763 20161 36688 97107 63340 91637 9943 36118 33017 82882 16323 97081 30112 25472 51621 9256 76965 58255 11289 40313 34569 60391 50809 28492 45779 250 56976 69210 62337 
13054 28391 46695 26998 34347 7846 38949 25431 11461 32610 42006 4296 13333 14758 38921 8827 25140 36555 14140 16549 42343 23004 10920 7852 26391 11821 41892 13887 21506 39043 22396 31825 34227 17960 36232 45495 41861 7913 18592 16280 10642 5203 38048 37450 35294 20560 20832 46100 47812 43836 21151 22044 21847 8673 39932 19290 20863 17239 27501 40560 16229 27357 12064 27669 40618 45404 45423 14652 24027 35907 21523 40989 32620 31050 43133 8435 33443 22265 14544 3212 48080 9388 16932 17371 22885 46444 28572 19981 13260 30450 19206 48729 34006 4361 27809 9425 5623 4271 26303 34220 3896 2286 35598 11401 41125 1638 38586 22873 11867 39273 34133 18182 7967 30572 21777 4393 22599 41290 6091 27282 42847 12571 25891 39898 6152 4318 39220 37907 10544 30201 28097 38105 40706 40952 6626 19319 45052 49130 41922 47079 41863 36523 21931 27685 18775 238 44908 21114 37156 26390 8279 33690 49192 15114 39233 40708 15709 44233 25209 33202 20843 26041 4537 21242 17134 29597 17978 6236 5015 35363 33095 34975 7165 28599 27900 35273 29758 37468 11102 6977 32701 33252 14385 22097 9143 10241 883 5780 28460 30415 10182 15425 20938 30550 35917 39589 15557 44191 32367 15374 
196
43106 87334 12258 59089 52663 20628 47583 79988 38641 73227 8842 3323 83481 59819 72552 26414 37695 23452 24077 95799 69769 31871 26186 9926 81144 51449 66924 66030 21389 8827 38073 41620 90422 47051 23946 47842 65259 60933 22516 91473 59069 85685 7960 62482 67968 55688 84289 5929 71027 89525 10809 81541 21371 96415 83215 88396 51031 67778 63859 94878 94586 26335 15629 61407 56879 60282 30281 92083 64146 72178 10027 57205 33169 33137 35162 23674 49172 34816 96709 52947 64092 19017 20939 74108 54977 37909 21998 86610 92660 26255 36310 39864 42732 32005 9437 18164 47512 69009 78167 21716 62920 57451 65583 66480 86481 10843 2422 70049 31758 89676 58061 54602 31802 63806 18003 51110 95759 72679 75113 43248 44808 63088 28779 62928 75330 23147 7011 97090 18196 539 41016 19156 29399 5143 40057 95579 64268 2726 60742 70043 36804 96364 69099 37039 71011 92162 40723 1458 18017 25019 1628 70497 85282 32569 39984 4557 51039 49587 84533 66989 3389 27959 65747 14291 11632 49410 59526 2694 54228 668 85951 11864 29466 40614 12516 46174 27718 52583 19850 64938 9098 7894 34120 34745 52860 72602 81645 30067 17187 74169 69040 88712 27879 49171 91308 91233 
3497 37431 31397 26867 28770 36273 44430 3585 32068 49908 18626 42004 22296 10325 2470 40000 18333 29823 8335 21001 37311 48405 580 46793 26798 11292 10298 40333 6844 29770 8577 11622 6588 32852 3343 5015 12621 8429 41549 24091 39672 16736 48831 21720 3079 40402 18838 43481 37107 37231 32686 16726 22931 2336 1273 3763 1418 29550 5463 46946 42509 38036 42218 1964 36968 8189 14702 10227 24354 23349 23375 11634 21352 40300 42192 35746 25925 31955 38337 3119 11557 2656 3616 16081 49614 21974 6615 18087 37091 40928 5798 47885 8959 39005 19934 39182 12077 31728 33962 9415 30365 18649 40454 43113 25558 43031 29281 13078 44478 31567 43891 31362 45987 1316 4431 11843 7919 39398 8494 42242 36079 31302 31850 41772 45001 31654 23651 38113 40030 29381 17169 8472 16008 23990 30556 48564 10069 16321 12693 28202 15472 15445 49521 14093 43022 42865 32573 45065 38715 28781 26780 10358 19277 41064 4143 42793 8068 33587 33900 10505 28964 36665 4861 32244 41496 26989 3526 42466 16824 18489 37115 10113 37738 45707 27102 10641 41147 14936 8926 28871 12209 12904 39247 6264 23416 32624 18062 49572 10432 9316 37419 28802 36995 23258 48272 41437 
197
49497 91781 42651 12315 36026 67508 76868 64725 49860 45331 60193 65123 28432 47840 42400 92866 73029 78608 39869 93659 86995 45657 13693 35703 79544 60394 25875 21273 26402 33114 91225 13181 49037 86859 40002 42756 69264 52738 81399 56314 4122 67590 429 93481 43899 37977 3529 9103 55178 16670 32108 35421 61978 69762 61630 64212 62072 33636 33487 89451 44019 59195 11902 38861 18449 32427 28089 80165 70980 40321 32837 36517 5047 87848 14387 28647 73648 83841 9573 67380 51326 92214 86284 2329 16985 26039 45932 29327 71889 46454 58666 6113 92874 22045 65624 36864 79479 8261 20643 32228 35578 56184 17559 3737 83535 55515 23261 58058 14484 44860 13177 97901 22486 24313 91557 13628 83990 48629 98998 99821 79178 51397 61904 92907 7356 23741 52756 14510 42535 7170 85618 95283 3229 13239 5563 33244 49037 31352 30187 73729 15734 10337 21209 88473 52817 46725 60204 3254 81967 56942 59499 96969 34354 51869 21732 58478 74396 55024 88343 31639 91539 66045 2280 8100 33871 42490 12633 90656 96310 28639 23588 92502 36376 5238 55229 374 42853 9594 45242 57686 79053 22943 87560 92922 42049 5490 65875 91703 674 60870 461 95567 38581 82736 41959 35949 65994 
43108 39152 1114 6531 25491 37445 9531 20462 23476 11434 18973 41044 49369 32104 35619 49600 20713 17647 39433 46482 101 47093 43313 20609 25792 1622 4010 26913 10148 4719 41281 19316 20914 48267 41946 5737 38323 17632 48340 4956 8785 15949 25819 21196 44012 39574 34954 23729 18552 44093 30799 28444 43358 37374 32716 42545 25397 32490 18439 16826 13355 47930 49030 3829 20924 39220 20530 4624 43264 10877 42122 46589 23444 36545 15574 40194 49477 30713 17310 17109 43225 30875 29872 29804 16114 7482 23598 18578 16570 34150 11967 1143 6826 3678 32890 23166 25503 16950 13715 12612 8551 11080 8361 24805 32226 46451 20958 47793 14987 45664 43344 13639 7774 46050 37662 15489 35886 9620 29320 47787 36829 22928 41759 41366 10832 28617 44866 37088 91 22257 43838 142 49280 43187 12863 45892 45395 18873 46714 23425 39462 13824 25384 28596 23523 40741 28900 31805 12278 46292 27437 30967 42647 10308 44322 26922 37734 21494 38682 20868 27056 6963 25328 20447 24331 7279 22830 26876 1188 28739 36560 29264 45590 19104 621 37948 9696 17470 31131 46797 12805 1042 36126 34427 40195 39149 23702 9963 32638 43777 39808 46686 1485 11869 2240 43970 11269 
199
24111 71402 4804 59283 908 81516 19801 35016 38462 66428 11492 16637 71957 65286 92858 79508 34289 20595 89394 63236 71838 70077 26035 13278 79951 57258 16991 107 59663 10563 55854 65644 26917 7411 82527 19506 58220 73286 43688 28096 86129 98614 74653 82937 1011 87966 7557 94525 21996 36438 3355 80697 65707 21499 91777 93026 1060 44697 46005 11649 17780 64450 50084 90018 27180 23291 14577 83924 38923 85504 34563 68291 68276 98559 68906 51780 89 78470 75378 60518 15118 51718 99809 92019 6764 35291 83222 48286 94607 62473 62592 96244 29106 7225 17235 72922 94074 44198 15599 86792 40492 19107 61417 36247 14073 13589 58774 51888 90518 64966 59771 78624 27170 30872 1592 75785 36828 18887 10321 2348 61811 98877 72239 51280 90412 70207 61721 87530 49292 42768 78803 83819 50767 24208 16369 28986 65691 5728 59638 83789 41099 12979 99359 64658 56941 20269 79026 90839 77959 28915 43989 60434 53036 89678 31289 61137 45915 94336 97578 97377 70431 42476 81003 9260 36151 1743 37323 58244 81704 17261 4932 46793 76557 16230 59273 23119 16263 56639 48209 89751 42569 12334 25263 44151 38418 19790 81377 59349 26835 83206 30901 10096 11967 48725 86005 18824 10510 3507 11340 
48438 37118 40568 35870 41915 11212 20692 49355 28293 39016 24136 7158 19893 8598 22204 24321 8067 5509 10240 37200 41803 109 21088 16267 13660 41691 36837 31642 48427 24790 10680 25606 16937 17431 15798 26046 35823 16428 28585 7202 4349 22168 48684 6969 31500 27705 34549 26204 1409 35983 35181 29162 22104 40929 20534 8001 25494 24750 22967 46279 48477 30372 17207 35898 36804 11502 27752 44702 22293 27166 24310 4136 3969 27532 49867 37224 41518 37870 29287 11034 26723 18493 6622 7700 33528 14182 16220 46246 20102 48633 47865 26392 34456 26324 40356 2776 23395 20392 14579 41829 10876 15514 19810 15667 13892 15130 32682 11417 26343 46681 38655 39871 24895 12731 2693 1457 47318 1957 49347 44790 11504 30293 9341 1986 25855 11630 36526 17648 40344 46883 34756 40586 48629 34610 18032 15633 9831 43596 17686 27484 1348 27286 40100 20351 2656 48710 13913 40800 12977 38066 26674 216 27812 10407 14815 7578 38742 24252 32443 3293 30959 35800 49848 40966 38866 26893 37433 45078 48878 13652 33344 22335 25699 7624 46927 12116 11942 33000 46362 48187 25168 20130 31472 40152 14392 31096 46562 43523 17444 33524 40 48000 16807 38230 40377 13009 5228 26845 46494 
200
52971 94662 17912 33886 32832 66158 97881 7045 38814 79386 56573 65377 95278 57641 24415 39312 48477 54027 56471 84822 52464 12221 45916 96557 51350 85817 12235 13383 42324 36680 69145 77547 2273 12116 24378 76061 96804 108 75055 82090 39393 28146 81221 96092 72471 85461 53503 33197 23905 35134 23008 7030 389 80646 69979 25276 72345 72667 72480 16131 37689 691 79365 92569 44311 27300 60436 55888 63717 35931 53139 74919 52801 94290 1630 30552 16406 79248 62048 25756 27624 27464 58159 50120 3615 46548 29337 24104 37518 63052 25453 9583 6742 93859 25230 26172 64414 38515 13284 7758 7936 68918 8364 23669 7992 83609 76328 8945 5548 74536 32058 98731 97730 99673 62372 60038 44703 84010 77317 72127 77248 62509 32820 38742 31068 94378 57422 68297 67667 88165 88190 21979 87439 48596 98890 20681 81366 8139 14663 25657 67505 21573 89647 76282 9674 9114 81882 62995 57055 49056 76909 68198 13970 96330 17185 75106 15399 64037 18685 51282 46142 5652 8495 3534 40443 4704 30335 38539 31264 34152 11309 86412 79060 90344 13461 36774 27580 88314 46529 44463 52624 76262 46989 96475 10846 70066 10806 1503 13612 10439 58021 94474 98184 36744 12236 37671 95535 17858 60877 97307 
42552 26022 17998 378 13397 13279 29801 3458 8412 4706 33271 21259 43917 14758 42263 16562 38082 688 27287 43373 49506 33914 19875 30928 42510 12879 8934 47392 39154 2626 34534 24170 24567 43315 32122 23367 16507 6576 23887 12522 41790 49926 13723 2018 46105 15925 34384 42238 44330 8247 39747 10299 31323 44333 4363 4791 18767 30393 33655 3834 28109 20292 48950 4051 23561 29870 1162 16858 44694 34051 9985 39807 18411 895 3368 1924 11223 36570 33661 13937 15288 30810 31202 44009 31126 30544 42214 42451 34289 21126 14769 31897 14957 34893 19106 23794 37757 1642 39871 48217 3764 14454 40084 27522 12721 4610 10082 28681 26070 20247 33838 7976 4693 38144 48169 44242 27468 6089 6458 47814 7980 48303 18781 44797 15473 43798 27029 33876 26398 30847 44040 35113 48135 4424 31727 16435 19490 28107 15512 13434 33699 13605 37936 41343 27303 44138 26418 27666 30277 12492 43573 46858 15289 17519 2869 46424 49944 3980 34366 33332 42524 43067 40836 15260 18078 27529 16554 15988 38275 2842 2367 35935 30257 4013 27279 38653 4081 18561 29700 14042 872 42946 45243 10721 34424 7533 14887 45177 27958 33160 49674 5405 45379 40166 32290 1515 778 5938 33830 7646 
198
965 37826 59644 34942 62637 87883 97963 58996 74685 16828 50947 50175 91715 8833 67441 15625 42985 1418 78655 17578 93022 21401 47070 8254 31912 5249 59489 4518 37343 38169 16926 29297 45360 44823 96346 30559 91235 19027 81492 17453 55634 9147 41236 38789 8606 88779 66973 82783 29501 27678 11158 99904 96977 31107 58207 81391 15415 50065 51000 52466 80364 96048 40562 91821 83740 38474 67027 1931 79042 7451 35113 76253 68710 47312 44479 62569 77520 94807 38913 89672 41410 25657 7086 94343 94132 97319 17260 52864 34854 17965 22860 67490 17946 42000 28036 54839 28798 73255 47687 45630 50677 39313 10620 51861 47082 42420 22698 10866 38490 28488 88311 81669 92846 25247 11123 72896 13927 43780 54816 21296 97147 98840 81435 10974 98339 80738 61511 39142 73461 90616 92888 40167 73167 46553 2210 88921 802 56411 82015 43147 47946 16227 3810 49921 24452 8875 91071 22255 2193 14004 42180 65553 58350 23296 44418 88425 22294 81306 37824 78535 27044 13947 44177 45970 34985 79183 65039 5580 95808 98685 20614 12616 16532 90683 92144 38123 83134 39200 27759 45509 90050 66184 41021 80893 25383 12313 78653 59545 72509 19024 1309 23214 49723 15785 57799 7087 41000 69142 
43982 14987 41343 38335 15887 47102 25408 8805 45369 40751 49355 29251 49916 36510 305 31907 21324 29291 27146 29969 1632 4986 28156 59 25181 6016 46901 25312 4921 48598 32608 11020 28294 48340 16073 14911 27229 22673 19422 17982 30727 20871 23634 35167 9578 26237 21070 40269 7545 47107 25868 28687 43067 24505 49436 34408 17989 23231 19203 25543 38578 12920 40973 9917 48692 196 19782 11291 7769 12077 20174 15966 8790 39924 8051 14190 45198 29726 5454 27123 10628 30147 34867 14467 335 41484 16710 47591 16003 9546 49136 26121 1718 8083 10801 45250 14692 10545 20917 45795 26980 22935 29258 29359 41807 20327 12788 42184 49378 36982 23646 17318 6319 42563 20174 24928 21893 44223 35665 35837 27682 41195 41390 33257 23530 17184 25396 9793 1321 30119 42438 16364 23013 30076 48570 36435 34117 39012 19768 41591 5068 45964 28576 1210 9624 7831 33725 12155 43469 6550 44778 39167 40831 618 45822 43557 26937 4414 16666 41483 24280 5136 41795 2314 18459 45937 46936 38960 47969 38745 32988 44507 25222 23259 38733 28480 32454 22870 30550 18578 8700 30879 38737 3810 5977 31282 31241 8746 41127 26339 17913 9679 4108 33215 40528 7824 15582 17116 
200
90678 34978 36057 11187 41474 50287 36160 6227 63347 76430 37372 3930 73322 8627 96187 469 40290 7168 31004 72656 34928 49683 96788 69592 18924 80772 88100 89206 1399 80907 85811 24347 17045 45241 7844 34019 3283 22853 32985 32140 32831 58446 13578 77083 69689 17873 53767 94649 99762 80889 28718 5192 96724 12743 60922 35514 36510 91123 66158 75160 95690 27132 55590 94803 72485 52502 58847 56988 75734 95013 11646 41055 99396 10043 40789 12408 2145 59136 97841 34671 22823 8516 93509 55590 23618 24293 32765 41615 56659 84232 86811 54377 80896 8858 72550 88398 69641 74677 73944 19643 1229 32510 93522 54802 12551 92759 71834 86306 87803 43479 86649 85970 99216 86772 25190 68218 35124 94420 41889 21298 69844 91988 45550 50978 83988 78096 34216 14911 99708 87275 21623 7314 12536 94835 81929 80514 373 13894 12067 55437 92915 98327 29290 56416 62808 63200 95388 5515 99914 92487 67027 24340 12321 29904 59288 28122 76056 3247 28477 54261 40882 73764 88445 26769 53754 88935 99928 74484 14053 95478 89990 61751 31215 31365 47862 9762 62134 62232 537 84127 6410 28582 4981 13551 10725 66018 92428 6566 85769 3557 85626 48992 84358 40726 77360 30570 18126 325 52519 79993 
37090 41878 4887 8499 44224 17347 45757 23255 44099 23333 8958 43014 11057 14343 37882 31676 39962 43401 30754 46745 16823 5671 14624 45078 36240 49067 31415 32428 43302 17203 35740 26681 26607 28213 18948 34399 201 22930 17551 49548 25467 28542 17621 46168 26575 36360 21559 26213 30310 15885 5169 49350 38730 40165 49293 9448 19685 8709 42432 32314 14362 15441 21623 18734 41530 21167 17874 38099 30890 26986 47359 19298 44747 20696 8253 6143 46747 29416 30185 39284 13858 49449 31394 29883 49017 15138 43767 32932 24051 22158 36506 37436 8242 37430 28022 19797 32622 5832 45913 2259 10201 17704 49453 19689 31713 19041 10623 14591 7542 17224 32485 23246 2789 14344 37137 15520 27107 38132 39879 28573 15975 39005 25450 40142 16423 23115 19546 11202 3461 37673 36281 48572 24439 37789 42895 31694 19563 33166 49986 42841 21489 45245 1699 46294 48590 37082 28362 27226 30353 24593 7389 20109 49700 44702 39754 17271 49011 7932 23528 16329 35884 37663 8389 41096 6815 20616 43659 19028 488 11965 4411 20125 24553 29259 17555 9084 38605 16197 25176 32877 594 11710 33318 37817 43280 38839 860 15989 17834 14596 29741 5791 10328 12966 12690 4585 17101 41654 38040 10968 
