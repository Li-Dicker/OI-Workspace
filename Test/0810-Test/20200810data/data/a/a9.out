Case #1: 13686969
Case #2: 13923824
Case #3: 13665332
Case #4: 14089166
Case #5: 13722494
Case #6: 13961838
Case #7: 14395016
Case #8: 13940896
Case #9: 13952449
Case #10: 15122455
